src/app/components/data-detail/data-detail.component.html

```diff
@@ -33,7 +33,7 @@ <h1 class="t-page-headline">Data</h1>
                 <div *ngIf="data?.producedBy != null">
                     <dt>Created in</dt>
                     <dd>
-                        <a [routerLink]="['/experiments', experimentId, 'timeline', data.producedBy]">
+                        <a [routerLink]="['/experiments', experimentId, 'timeline', data.producedBy]" queryParamsHandling="merge">
```

Comment:
if no query param is updated (which is the case for both changed links in this file), then is there a difference between merge and preserve? If not, then consistently use one strategy for all these instances.

Reply:
Both strategies work the same way in that case. I chose `'merge'` in case future updates add new parameters.